PowerUp/app/src/main/java/powerup/systers/com/Game.java

```diff
@@ -47,13 +54,13 @@ public void onCreate(Bundle savedInstanceState) {
 		answers = new ArrayList<>();
 		goToMap = (Button) findViewById(R.id.continueButtonGoesToMap);
 		replay = (Button) findViewById(R.id.redoButton);
-		
-		/* Code for putting the avatar. Good Luck! GSoC-2016!"
-		eyeView = (ImageView) findViewById(R.id.eyeImageView);
-		faceView = (ImageView) findViewById(R.id.faceImageView);
-		hairView = (ImageView) findViewById(R.id.hairImageView);
-		clothView = (ImageView) findViewById(R.id.clothImageView);
-		
+
+		// Code for putting the avatar. Good Luck! GSoC-2016!"
+		eyeView = (ImageView) findViewById(R.id.eyeView);
```

Comment:
Probably `eyeImageView` is a better name for both the resource and the variable. The same for the other ones